crates/cheatcodes/spec/src/vm.rs

```diff
@@ -697,6 +697,10 @@ interface Vm {
     #[cheatcode(group = Testing, safety = Safe)]
     function zkUsePaymaster(address paymaster_address, bytes calldata paymaster_input) external pure;
 
+    /// Marks the contract to be injected as a factory dependency in the next transaction
+    #[cheatcode(group = Testing, safety = Safe)]
+    function zkUseFactoryDep(string calldata name) external pure;
+
```

Comment:
Please generate the json interface https://github.com/matter-labs/foundry-zksync/blob/main/docs/dev/cheatcodes.md#json-interface